lib/iron_bank/resources/notification.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module IronBank
+  module Resources
+    class Notification < Resource
+      with_schema
+
+      with_one :communication_profile
+
+      ENDPOINT = "/notifications/notification-definitions"
+
+      def self.find(id)
+        raise IronBank::NotFoundError unless id
```

Comment:
do we need this line? is this the pattern we have for the other resources? i would expect a `nil` when used as `id` will generate the same.